docker/try_embedded.bat

```diff
@@ -0,0 +1,15 @@
+@echo off

```

Comment:
Is it possible to merge this somehow with the try_embedded.sh script?

Reply:
@tib, Windows user could use gitbash to use the sh scripts, but my experience show that not everyone is actually using git! Those scripts are are just drop in for anyone to use it so you may have a user downloading the zip of the repo and run the script. Plain and simple. 

Another approach is to use docker-compose (I am not use to it, will check it out).

Reply:
I'd prefer to use docker-compose if possible. ;)

Reply:
Done... So in the folder use:
`docker-compose up`
`docker-compose -f mysql.yml up`
`docker-compose -f postgres.yml up`